controllers/watermarkpodautoscaler_controller.go

```diff
@@ -302,6 +303,16 @@ func (r *WatermarkPodAutoscalerReconciler) reconcileWPA(ctx context.Context, log
 	return r.updateStatusIfNeeded(ctx, wpaStatusOriginal, wpa)
 }
 
+func requeueAfterForWPAErrors(err error) time.Duration {
+	// We don't expect this error to be recovered after 1s, so define a longer
+	// delay.
+	if strings.Contains(err.Error(), scaleNotFoundErr) {
+		return 10 * time.Second
```

Comment:
let's specify these two return values as const (here and l:313).

Reply:
Changed :+1: